source/content/guides/multisite/03-config.md

```diff
@@ -120,9 +143,32 @@ Complete the steps below after spinning up a new WPMS site from the correct Cust
 1. Locate the `/* That's all, stop editing! Happy Pressing. */` line, and add the following code above this line to enable the WPMS configuration.
 
   ```php:title=wp-config.php
+  $hostname = 'www.yourdomain.com'; // The domain of the network. Use as a fallback if $_SERVER['HTTP_HOST'] is not available.
+  if ( !empty( $_ENV['PANTHEON_ENVIRONMENT'] )) {
+    $site_name = $_ENV['PANTHEON_SITE_NAME'];
+    // Override $hostname value as needed.
+    switch ( $_ENV['PANTHEON_ENVIRONMENT'] ) {
+      case 'live':
+        // Fall back to the default $hostname if $_SERVER['HTTP_HOST'] is not available.
+        $hostname = isset( $_SERVER['HTTP_HOST'] ) ? $_SERVER['HTTP_HOST'] : $hostname;
+        break;
+      case 'test':
+        $hostname = isset( $_SERVER['HTTP_HOST'] ) ? $_SERVER['HTTP_HOST'] : 'test-' . $site_name . '.pantheonsite.io';
+        break;
+      case 'dev':
+        $hostname = isset( $_SERVER['HTTP_HOST'] ) ? $_SERVER['HTTP_HOST'] : 'dev-' . $site_name . '.pantheonsite.io';
+        break;
+      case 'lando':
+        $hostname = $site_name . '.lndo.site';
+        break;
+      default:
+        $hostname = $_ENV['PANTHEON_ENVIRONMENT'] . '-' . $site_name . '.pantheonsite.io';
+        break;
+    }
+  }
   define( 'MULTISITE', true );
-  define( 'SUBDOMAIN_INSTALL', false ); // Set this to TRUE for Subdomains
-  define( 'DOMAIN_CURRENT_SITE', $_SERVER['HTTP_HOST'] );
+  define( 'SUBDOMAIN_INSTALL', <?php echo $subdomain_install ? 'true' : 'false'; ?> );
```

Comment:
no sure I follow what is going on here? What is this inline conditional?

Reply:
I copy/pasted a large part of this from the WPCM repository. This may have been from that and it might not be a thing we want/need here.

Reply:
yeah this was copy/pasta from the mu-plugin where we're detecting something from a previous step. I've removed.